ui/src/snack/browserNotification.ts

```diff
@@ -19,10 +19,17 @@ export function notifyNewMessage(msg: IMessage) {
     const notify = new Notify(msg.title, {
         body: removeMarkdown(msg.message),
         icon: msg.image,
+        silent: true,
         notifyClick: closeAndFocus,
         notifyShow: closeAfterTimeout,
-    });
+    } as any);
     notify.show();
+
+    if (msg.priority >= 4 && !Notify.needsPermission) {
```

Comment:
I think the audio should be played regardless if the user allows notifications. Furthermore, I think the audio should be started inside the reaction.ts
```typescript
stores.wsStore.listen((message) => {
    stores.messagesStore.publishSingleMessage(message);
    Notifications.notifyNewMessage(message);
    if (message.priority >= 4) {
        const src = 'static/notification.ogg';
        const audio = new Audio(src);
        audio.play();
    }
});
```